src/komponenter/Adressesperre/HemmeligAdresseVarsel.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Alert, Heading, Loader } from '@navikt/ds-react';
+import { useAvtaleKreverAktsomhet } from '@/services/use-rest';
+import { useParams } from 'react-router-dom';
+
+interface props {
+    avtaleId?: string;
+}
+
+const HemmeligAdresseVarsel = () => {
+    const { avtaleId } = useParams<{ avtaleId: string }>();
+    const { data: avtaleKreverAktsomhet } = useAvtaleKreverAktsomhet(avtaleId);
+
+    return avtaleKreverAktsomhet ? (
+        <Alert variant={'warning'}>
+            <Heading spacing size="small" level="3">
+                Hemmelig adresse
+            </Heading>
+            Denne personen har hemmelig adresse og du må derfor utvise aktsomhet.
+        </Alert>
+    ) : null;
```

Comment:
Vurder kanskje å ha en *early-return* istedenfor ternary , det er litt enklere å lese.

```suggestion
   if (avtaleKreverAktsomhet) {
       return null;
   }

    return (
        <Alert variant={'warning'}>
            <Heading spacing size="small" level="3">
                Hemmelig adresse
            </Heading>
            Denne personen har hemmelig adresse og du må derfor utvise aktsomhet.
        </Alert>
    );
```